### PR TITLE
Better error messages and minor styling fixes

### PR DESCRIPTION
**Pull request by axelboc**
Various tweaks to address issues identified notably in https://github.com/silx-kit/h5web/discussions/1668

### CODE REVIEW

File: packages/app/src/explorer/Explorer.module.css [hunk 1]
```diff
@@ -64,6 +64,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  line-height: 1.5; /* fix cropping of ascenders and descenders */
```
Comment (axelboc) [inline packages/app/src/explorer/Explorer.module.css:67]: image 👉  image

File: packages/app/src/metadata-viewer/MetadataViewer.module.css [hunk 2]
```diff
@@ -9,9 +9,12 @@
   margin-top: 1rem;
 }
 
+.title {
+  text-transform: capitalize;
+}
+
 .table th {
   font-weight: 500;
-  text-transform: capitalize;
```
Comment (axelboc) [inline packages/app/src/metadata-viewer/MetadataViewer.module.css:14]: image 👉 image

File: packages/app/src/metadata-viewer/RawInspector.tsx [hunk 3]
```diff
@@ -16,8 +18,8 @@ function RawInspector(props: Props) {
         {JSON.stringify(
           entity,
           (key, value) => {
-            // Bypass cyclic dependencies
-            return key !== 'parents' && key !== 'children' ? value : undefined;
+            // Bypass `value` and cyclic dependencies
+            return KEYS_TO_SKIP.has(key) ? undefined : value;
```
Comment (axelboc) [inline packages/app/src/metadata-viewer/RawInspector.tsx:22]: image 👉 image

File: packages/app/src/explorer/Explorer.module.css [hunk 4]
```diff
@@ -61,7 +61,9 @@
 }
 
 .name {
-  overflow: hidden;
+  align-self: stretch; /* prevent cropping of ascenders and descenders */
+  display: flex;
+  align-items: center;
```
Comment (axelboc) [inline packages/app/src/explorer/Explorer.module.css:66]: image 👉  image

File: packages/app/src/metadata-viewer/RawInspector.tsx (hunk 3 above)
Comment (joaonizer) [reply packages/app/src/metadata-viewer/RawInspector.tsx:22]: 👈

File: packages/app/src/explorer/Explorer.module.css [hunk 5]
```diff
@@ -64,6 +64,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  line-height: 1.5; /* fix cropping of ascenders and descenders  */
```
Comment (axelboc) [inline packages/app/src/explorer/Explorer.module.css:67]: image 👉  image

Comment (axelboc) [reply packages/app/src/explorer/Explorer.module.css:67]: The alignment of the text with the arrow changes very slightly, but I think for the better.

**Review by loichuder — APPROVED** [review]

**Pull request merged**